tests/rector_bootstrap.php

```diff
@@ -0,0 +1,10 @@
+<?php
+declare(strict_types=1);
```

Comment:
@TomasVotruba this file looks really similar to `tests/bootstrap.php` couldn't we reuse it?

Reply:
We could and we did, but the bypass final removes all `final` keywords :D

Reply:
ok so rector will try to add it back 😄 
makes sense, let's merge it then!! 💪